### PR TITLE
feat: Added OSCReceiverFragment

### LINKED ISSUE #192: Missing OSCReceiver Actvity
**Actual Behaviour**
Missing OSCReceiver Actvity. The activity/fragment needs to be developed in accordance with the base class.

**Expected Behaviour**
The activity should be present.

**Would you like to work on the issue?**
Yes, I will be working on it.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #192 

**Changes**: Added OSCReceiverFragment. Made the layout and the Fragment class.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by jddeep** [comment]
I could not implement methods/functionality to this because #190 is still unmerged, Will be doing that once the former is merged.

**Comment by jddeep** [comment]
Hence, it is a WIP.

**Comment by CloudyPadmal** [comment]
@jddeep you're good to go now. #190 merged.

**Comment by jddeep** [comment]
@CloudyPadmal Please review.

File: app/src/main/java/io/neurolab/network/OSCReceiverFragment.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                oscReceiver = new OSCReceiver(Integer.parseInt(String.valueOf(portField.getText())), listenerList);
+            } catch (SocketException e) {
+                e.printStackTrace();
+            }
+        });
+    }
+
+    private void getItemList() {
+        listenerList = new ArrayList<>();
+        for(int i = 0;i<listenerTextList.getChildCount();i++){
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/network/OSCReceiverFragment.java]: Please reformat the code before committing

File: app/src/main/res/layout/fragment_oscreceiver.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/fragment_oscreceiver.xml:1]: This file also, reformat

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/neurolab/network/OSCReceiverFragment.java (hunk 1 above)
Comment (jddeep) [reply app/src/main/java/io/neurolab/network/OSCReceiverFragment.java]: Done

File: app/src/main/res/layout/fragment_oscreceiver.xml (hunk 2 above)
Comment (jddeep) [reply app/src/main/res/layout/fragment_oscreceiver.xml:1]: Done

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**